ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml

```diff
@@ -1,21 +1,13 @@
 ﻿@model GeneralInformationViewModel
 @{
-	var kw = WidthOverride.TwoThirds;
-	var distance = "";
+		var kw = WidthOverride.TwoThirds;
```

Comment:
Unsure how this looks in VS, but the indentation appears to have an extra tab. Please double check.